src/DotLiquid/Tags/AddFilters.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text.RegularExpressions;
+using DotLiquid.Exceptions;
+using DotLiquid.Util;
+
+namespace DotLiquid.Tags
+{
+    /// <summary>
+    /// Adds filters in a whitelisted class to the current context.
+    ///
+    /// {%- addfilters 'ShopifyFilters' -%}
+    /// </summary>
+    public class AddFilters : Tag
+    {
+        private static readonly Regex Syntax = R.B(R.Q(@"\s*([""'].+[""'])\s*"));
+
+        private static readonly IDictionary<string, Type> FILTER_WHITELIST = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase);
+        static AddFilters()
+        {
+            WhitelistFilter(typeof(ShopifyFilters));
```

Comment:
Would this be easier to find if it was in Liquid.cs together with StandardFilters?
```
Template.RegisterFilter(typeof(StandardFilters));
AddFilters.WhitelistFilter(typeof(ShopifyFilters)); //Don't register it but whitelist it for template authors
```
This way when my PR merges it can also add the ExtendedFilters to the whitelist in the same place.

Reply:
Moved